www/front_src/src/Navigation/Sidebar/Menu/CollapsibleItems.tsx

```diff
@@ -0,0 +1,366 @@
+        enter={false}
+        exit={false}
+        in={isCollapsed}
+        ref={collapsRef}
+        timeout={0}
+        onMouseLeave={onLeave}
+      >
+        {data?.map((item, index) => {
+          const hover =
+            isItemHovered({
+              currentPage: item,
+              levelTitle: levelName,
+              navigationItem: selectedNavigationItems,
+            }) || equals(hoveredIndex, index);
+          const mouseEnterItem = (e: React.MouseEvent<HTMLElement>): void =>
+            hoverItem({ currentPage: item, e, index });
+
+          return (
+            <List
+              disablePadding
+              key={item.label}
+              subheader={
+                isSubHeader && (
+                  <ListSubheader
+                    disableGutters
+                    disableSticky
+                    className={classes.subHeader}
+                  >
+                    {item.label}
+                  </ListSubheader>
+                )
+              }
+            >
+              {isSubHeader ? (
+                isArrayItem(item?.children) &&
+                item?.children?.map((content, ind) => {
+                  const nestedIndex = getNestedIndex({
+                    childIndex: ind,
+                    content: data,
+                    itemIndex: index,
+                  });
+                  const nestedHover =
+                    isItemHovered({
+                      currentPage: content,
+                      levelTitle: levelName,
+                      navigationItem: selectedNavigationItems,
+                    }) || equals(hoveredIndex, nestedIndex);
+
+                  const mouseEnterContent = (
+                    e: React.MouseEvent<HTMLElement>,
+                  ): void =>
+                    hoverItem({ currentPage: content, e, index: nestedIndex });
+
+                  return (
+                    <MenuItems
+                      data={content}
+                      hover={nestedHover}
+                      isOpen={nestedIndex === hoveredIndex}
+                      key={content.label}
+                      onClick={
+                        !isArrayItem(item?.groups)
+                          ? (): void => onClick(content)
+                          : undefined
+                      }
+                      onMouseEnter={mouseEnterContent}
+                    />
+                  );
+                })
+              ) : (
+                <MenuItems
+                  data={item}
+                  hover={hover}
+                  isOpen={index === hoveredIndex}
+                  onClick={
+                    !isArrayItem(item?.groups)
+                      ? (): void => onClick(item)
+                      : undefined
+                  }
+                  onMouseEnter={mouseEnterItem}
+                />
+              )}
+
+              {Array.isArray(item?.groups) &&
+              item.groups.length > 1 &&
```

Comment:
You can extract that `Array.isArray(item?.groups) && item.groups.length > 1` into a specific variable